src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -9,68 +9,204 @@ import { localize } from 'vs/nls';
+						</span>
+					</div>
+					<div class="tool_tip__container" id="tool_tip__container--narrow">
+						<a aria-haspopup="true" class="preview_link" tabindex="0" id="preview_link--narrow" name="previewNarrow"><p>Preview</p><i class="icon--info themed_icon"></i></a>
+					</div>
 				</div>
-				<div class="section recent">
-					<h2 class="caption">${escape(localize('welcomePage.recent', "Recent"))}</h2>
-					<ul class="list">
-						<!-- Filled programmatically -->
-						<li class="moreRecent"><a href="command:workbench.action.openRecent">${escape(localize('welcomePage.moreRecent', "More..."))}</a><span class="path detail if_shortcut" data-command="workbench.action.openRecent">(<span class="shortcut" data-command="workbench.action.openRecent"></span>)</span></li>
-					</ul>
-					<p class="none detail">${escape(localize('welcomePage.noRecentFolders', "No recent folders"))}</p>
+				<div id="preview_modal" class="modal" aria-modal="true" aria-hidden="true">
+					<div class="modal_content">
+						<span class="close_icon">x</span>
+						<h3 tabindex="0" class="preview_modal__header">${escape(localize('welcomePage.previewHeader', "This page is in preview"))}</h3>
+						<p tabindex="0" class="preview_modal__body">${escape(localize('welcomePage.previewBody', "Preview features introduce new functionalities that are on track to becoming a permanent part the product. They are stable, but need additional accessibility improvements. We welcome your early feedback while they are under development."))}</p>
+					</div>
 				</div>
-				<div class="section help">
-					<h2 class="caption">${escape(localize('welcomePage.help', "Help"))}</h2>
-					<ul>
-						<li><a href="https://aka.ms/get-started-azdata">${escape(localize('welcomePage.gettingStarted', "Getting started"))}</a></li>
-						<li><a href="https://aka.ms/azuredatastudio">${escape(localize('welcomePage.productDocumentation', "Documentation"))}</a></li>
-						<li><a href="https://github.com/Microsoft/azuredatastudio/issues/new/choose">${escape(localize('welcomePage.reportIssue', "Report issue or feature request"))}</a></li>
-						<li><a href="https://github.com/Microsoft/azuredatastudio">${escape(localize('welcomePage.gitHubRepository', "GitHub repository"))}</a></li>
-						<li><a href="https://aka.ms/azuredatastudio-releasenotes">${escape(localize('welcomePage.releaseNotes', "Release notes"))}</a></li>
-					</ul>
+				<div class="ads_homepage__section section header hero">
+					<div class="row start">
+						<div class="header__top_nav">
+							<div class="flex">
+								<div class="icon"></div>
+								<div>
+									<h1>Azure Data Studio</h1>
```

Comment:
localize?

Reply:
This should really just be filled in from the product service like we already do here https://github.com/microsoft/azuredatastudio/blob/42cad885f60ef9afde6d394736b97db166cb1fe2/src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts#L254

Reply:
fixed